src/sentry/static/sentry/app/views/projectInstall/createProject.jsx

```diff
@@ -332,11 +319,11 @@ class CreateProject extends React.Component {
                for your API server and frontend client.`
             )}
           </HelpText>
-          {hasIssueAlertOptionsEnabled && (
+          {this.issueAlertOptionsEnabled && (
             <PageHeading withMargins>{t('Choose a platform')}</PageHeading>
           )}
           <PlatformPicker platform={platform} setPlatform={this.setPlatform} showOther />
-          {hasIssueAlertOptionsEnabled && (
+          {this.issueAlertOptionsEnabled && (
```

Comment:
getters aren't cached right? Perf impact on this getter is negligible (I can't even tell if all of these are in the same method either), but something to be aware of.

Reply:
Correct it is not cached.